src/Anonymization/Anonymizer/AbstractAnonymizer.php

```diff
@@ -99,6 +99,21 @@ protected function getSalt(): string
         return $this->options->get('salt') ?? Anonymizator::generateRandomSalt();
     }
 
+    /**
+     * Validate Options given to the Anonymizer.
+     *
+     * Override this method for your needs, for example validate that
+     * an option `foo` is given with the correct type.
+     *
+     * This method is launch before the beginning of the anonymization process.
+     *
+     * If you override this method, you must call `parent::validateOptions()`
+     * at the begining of your implementation.
+     *
+     * @throws \Exception if given options are not valide.
+     */
+    public function validateOptions(): void {}
```

Comment:
```suggestion
    public function validateOptions(): void
    {
    }
```

Reply:
CS fixer will not be happy, it puts empty braces in a single line.

Reply:
Ah! I thought it was the opposite. My suggestion aimed to stick PHP CS fixer rules :sweat_smile: